src/handlers.py

```diff
@@ -22,6 +25,8 @@ async def play(ctx: commands.context.Context, url: str):
 
     queue = MuzlagQueue()
     await queue.push(channel.id, url)
+    url_title = get_yt_title(url)
```

Comment:
Better do 
`player = function player_factory(url)`, then you can get title from player
and then push player in queue